### PR TITLE
1209 balance changes

### PR DESCRIPTION
**Pull request by wang429**
BUFF: burning - amount reduced by down from 50% to 33% (1/2 to 1/3)
BUFF: burning - healing prevented (before, reduced by 50%)
BUFF: burning - Time Eater's healing is now prevented if it was on fire

addresses #230

### CODE REVIEW

File: src/main/java/stsjorbsmod/patches/BurningTimeEaterPatch.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import stsjorbsmod.powers.BurningPower;
+
+import java.util.Iterator;
+
+public class BurningTimeEaterPatch {
+    private static final String UI_ID = JorbsMod.makeID(BurningTimeEaterPatch.class);
+    private static final String[] TEXT = CardCrawlGame.languagePack.getUIString(UI_ID).TEXT;
+
+    @SpirePatch(clz = RemoveDebuffsAction.class, method = "update")
+    public static class RemoveDebuffsAction_Update_BurningTimeEater {
```
Comment (dbjorge) [inline src/main/java/stsjorbsmod/patches/BurningTimeEaterPatch.java]: I think you could probably simplify and avoid a patch for this by instead adding something like this to `BurningPower`:

```java
    @Override
    public void onRemove() {
        // This makes the heal prevention affect Time Eater's "cleanse and heal simultaneously" turn
        // despite the cleanse technically happening first.
        for (AbstractGameAction a : AbstractDungeon.actionManager.actions) {
            if (TimeEater.ID.equals(this.owner.id) && a instanceof HealAction && a.target == this.owner) {
                a.amount = this.onHeal(a.amount);
            }
        }
    }
```

File: src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Keyword-Strings.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [
   {
     "NAMES": ["burning"],
-    "DESCRIPTION": "Burning creatures take damage at the start of their turn and have healing reduced by 50%. Each turn, Burning is reduced by half."
+    "DESCRIPTION": "Burning creatures take damage at the start of their turn and have healing prevented. Each turn, Burning is reduced by a third."
```
Comment (dbjorge) [inline src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Keyword-Strings.json]: I feel like "have healing prevented" sounds a little ambiguous about whether it prevents them from healing others; maybe "cannot be healed" would be more clear, more like how you've update the power strings?

File: src/main/java/stsjorbsmod/patches/BurningTimeEaterPatch.java (hunk 1 above)
Comment (wang429) [reply src/main/java/stsjorbsmod/patches/BurningTimeEaterPatch.java]: Oh yeah... forgot to check the overrideable methods on AbstractPower... Yeah I'll take a look and see.
Another thing I was considering was to simply remove the healing action all together. Heal for 0 doesn't have the animation as far as I can tell.

File: src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Keyword-Strings.json (hunk 2 above)
Comment (wang429) [reply src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Keyword-Strings.json]: Sure, that makes sense.

**Pull request merged**